pkg/kubectl/cmd/run.go

```diff
@@ -254,6 +276,22 @@ func Run(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer, cmd *cob
 	return nil
 }
 
+func contains(resourcesList map[string]*unversioned.APIResourceList, gv unversioned.GroupVersion, name string) bool {
```

Comment:
Also, how about a `TODO` to make this more library-esq.  It looks like it would have general utility as a layer on top of a discovery client.